.vscode/settings.json

```diff
@@ -1,7 +1,8 @@
-{
-    "files.exclude": {
-        "dist/lib.csharp/**": true,
-        "src.csharp/**": true,
-        "node_modules/**": true
-    }
+{
+    "files.exclude": {
+        "dist/lib.csharp/**": true,
+        "src.csharp/**": true,
+        "node_modules/**": true
+    },
+    "files.eol": "\n"

```

Comment:
I keep having issues with VS Code together with git `core.autocrlf=false` (resulting in line endings being switched when i edit files). I added this believing it would fix stuff, but it seems I have more fixing to do on my end. Reverting this.